src/Domain/Infra/DependencyInjection/Compiler/DoctrineObjectFieldMappingPass.php

```diff
@@ -30,6 +31,7 @@ public function __construct(string $tagName = 'msgphp.doctrine.object_field_mapp
     public function process(ContainerBuilder $container): void
     {
         $mappings = [];
+        $config = MappingConfig::fromArray($container->getParameter('msgphp.doctrine.mapping_internal_config'));
```

Comment:
construct the same as runtime: `new MappingConfig($c->getParam(...), $c->getParam(...))`